### ISSUE TITLE
[UX] Verbose log messages from experimental compilation config

### ISSUE THREAD

**Issue by tqchen** [apache/tvm#9487]
reported by @masahi 

After updating to the latest main, I'm seeing too many LOG(INFO) from src/target/compilation_config.cc . Below is the log emitted when compiling a single dense op.
```
[15:51:23] /home/masa/projects/dev/tvm/src/target/compilation_config.cc:87: Using the unique primitive target Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0}) of device type 1 for the host target
[15:51:23] /home/masa/projects/dev/tvm/src/target/compilation_config.cc:141: Using the device type 1 of the unique primitive target as the default device type for all primitive operations
[15:51:23] /home/masa/projects/dev/tvm/src/target/compilation_config.cc:225: Target Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0}, host=Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0})) of device type 1 is available for primitives
[15:51:23] /home/masa/projects/dev/tvm/src/target/compilation_config.cc:228: Using default primitive scope SEScope(device_type=1, virtual_device_id=0, target=Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0}, host=Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0})))
[15:51:23] /home/masa/projects/dev/tvm/src/target/compilation_config.cc:229: Using host scope SEScope(device_type=1, virtual_device_id=0, target=Target(kind='llvm', keys={'cpu'}, attrs={"link-params': (bool)0}))

````

**Comment by tqchen** [comment]
was introduced by https://github.com/apache/tvm/pull/9313, sorry that I missed it when merging. going to send a hotfix

**Comment by tqchen** [comment]
https://github.com/apache/tvm/pull/9486

**Comment by junrushao1994** [comment]
Closed via #9486
